### ISSUE TITLE
How do I import an npm module whatever

### ISSUE THREAD

**Issue by villander** [emberjs/guides#2223]
how do I import an npm module whatever (AMD, commonJS, UMD) in ember-cli?

We should better document this, and ensure the community's ergonomics by putting a third-party library in your app, and adding that into an addon (which is even more painful)

**Comment by pbishop16** [comment]
@villander For now, this section in the ember-cli guides should provide you most of what you are looking for:
 [ember-cli User-guide: Managing Dependencies](https://ember-cli.com/user-guide/#managing-dependencies)

**Comment by villander** [comment]
No! You are mistaken. It is not very clear how to import npm modules be they AMD, CMJ and etc in ember apps. And even modules made in TS.

We had a long discussion on slack and then we decided to open this issue with the @Turbo87

**Comment by pbishop16** [comment]
@villander Ah, okay understood. Thanks for the feedback. I will standby for further insight.
